core/src/main/java/utilcalc/core/model/output/WaterHeatingBasicPart.java

```diff
@@ -0,0 +1,14 @@
+package utilcalc.core.model.output;
+
+import static utilcalc.core.utils.Util.ensureNonNull;
+
+import java.math.BigDecimal;
+
+public record WaterHeatingBasicPart(
+        BigDecimal numberOfMonths, BigDecimal monthlyCost, BigDecimal totalAmount) {
+    public WaterHeatingBasicPart {
+        ensureNonNull(numberOfMonths, "numberOfMonths");
```

Comment:
Jeste tady bude muset byt DateRange, podobne jako ve WaterFee. Na obrazku to sice nebylo, ale pokud dojde ke zmene ceny v uctovacim obdobi, tak ve vystupu bude vic radku a bylo by dobre u nich napsat od-do.